framework/commercetools/lib/normalize.ts

```diff
@@ -0,0 +1,180 @@
+import type {
+  CommercetoolsProducts,
+  Product,
+  ProductVariant,
+  CommercetoolsProductVariant,
+  CommerceToolsProductPrice,
+  ProductPrice,
+} from '../types/product'
+import type {
+  Cart,
+  CommercetoolsCart,
+  CommercetoolsLineItems,
+  LineItem,
+} from '../types/cart'
+
+import type {
+  CommercetoolsBrands,
+  CommercetoolsCategory,
+  Category,
+  Brand,
+} from '../types/site'
+
+import { arrayToTree } from './array-to-tree'
+
+function normalizeVariants(
+  variants: CommercetoolsProductVariant[],
+  published: boolean
+): ProductVariant[] {
+  return variants.map((variant) => {
+    return {
+      id: variant.id,
```

Comment:
missing SKU here

Reply:
This is the ProductVariant representation of vercel-commerce, not Commercetools and hence that's why the variant is not here (and it's at Product level as mentioned in other comment)